roles/openshift_facts/library/openshift_facts.py

```diff
@@ -493,7 +498,7 @@ def get_current_config(facts):
         # anything from working properly as far as I can tell, perhaps because
         # we override the kubeconfig path everywhere we use it?
         # Query kubeconfig settings
-        kubeconfig_dir = '/var/lib/openshift/openshift.local.certificates'
+        kubeconfig_dir = '/var/lib/{{ product_type }}/openshift.local.certificates'
```

Comment:
@detiber I agree these changes were made thinking that AE and OSE are going have different naming structure. But things are changed now it seems.